.github/workflows/chroma-test.yml

```diff
@@ -23,7 +23,8 @@ jobs:
                    "chromadb/test/property/test_cross_version_persist.py",
                    "chromadb/test/property/test_embeddings.py",
                    "chromadb/test/property/test_filtering.py",
-                   "chromadb/test/property/test_persist.py"]
+                   "chromadb/test/property/test_persist.py",
+                   "chromadb/test/utils/test_embedding_functions.py"]
```

Comment:
Please ignore this also by adding another --ignore-glob to the first testfile as this will be run twice.

Reply:
Done..